bin/GameIndex.yaml

```diff
@@ -12676,9 +12677,6 @@ SLES-53446:
   name: "Arcade USA"
   region: "PAL-E"
   compat: 5
-SLES-53450:
```

Comment:
Why was this removed?

Reply:
Yeah that's going to break the parsing of the file.

Reply:
Sces-53450 is the correct serial. Sles-53450 is wrong. http://redump.org/disc/12328/

Reply:
@Mrlinkwii Can you confirm with Red what the correct serial is?

Reply:
done

Reply:
okay, I see the entry is already in there, Do you want to correct the title for the entry that's already in there, Red?

Reply:
I'll do a bigger PR later and include that as well.